rbac/management/audit_log/model.py

```diff
@@ -95,13 +95,10 @@ def get_resource_item(self, r_type, request, *args, **kwargs):
             # TODO: update for permission related items
             return None
 
-        elif r_type == "principal":
-            principal_object = get_object_or_404(Principal, username=request.user.username, tenant=verify_tenant)
-            return principal_object.id, principal_object.username
-
     def log_create(self, request, resource):
```

Comment:
@Ellen-Yi-Dong Checking the PR_check failure we have a constraint violation in the `management_auditlog`.
Using a fresh database, I tried to create a new role and got this error due to not have the user requester as a principal yet:
```
{
  "errors": [
    {
      "detail": "insert or update on table \"management_auditlog\" violates foreign key constraint \"management_auditlog_principal_id_512ce4d0_fk_managemen\"\nDETAIL:  Key (principal_id)=(51834776) is not present in table \"management_principal\".\n",
      "source": "role",
      "status": "400"
    }
  ]
}
```

Maybe we can check and add the user requester as a principal inside the `log_create` method.